eZ/Publish/Core/Limitation/ParentContentTypeLimitationType.php

```diff
@@ -234,4 +234,20 @@ public function valueSchema()
     {
         throw new \eZ\Publish\API\Repository\Exceptions\NotImplementedException(__METHOD__);
     }
+
+    /**
+     * @param \eZ\Publish\API\Repository\Values\ValueObject $contentInfo
+     * @return \eZ\Publish\API\Repository\Values\Content\Location[]
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     */
+    private function loadParentLocations(ValueObject $contentInfo): array
+    {
+        $locations = $this->persistence->locationHandler()->loadLocationsByContent($contentInfo->id);
+        $parentLocations = [];
+        foreach ($locations as $location) {
```

Comment:
```suggestion
        $this->persistence->locationHandler()->loadList()
```
I think that using this new method could be more efficient